sor/src/main/java/com/bazaarvoice/emodb/sor/db/test/InMemoryDataReaderDAO.java

```diff
@@ -300,7 +301,7 @@ public synchronized void deleteDeltasOnly(Table table, String key, UUID compacti
     }
 
     public synchronized void addCompactionOnly(Table table, String key, UUID compactionKey, Compaction compaction,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Method has 9 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS00107)